br-webui/views/system.liquid

```diff
@@ -277,7 +272,7 @@ socket.on('cpu stats', function(data) {
 			</div>
 			<div class="panel-body">
 				<h4>Version: </h4>
-				<span id="companion version">Companion Version</span>
```

Comment:
Can you use the old id ? "cv" is not explicit and will need more work in the future to someone to discover what cv means.

Reply:
If i dont use a different id, the companion version is not getting displayed on the system page, only in the header

Reply:
There should be no changes to this file.